spring-cloud-skipper-docs/src/main/asciidoc/command-reference.adoc

```diff
@@ -231,7 +231,7 @@ The number of revisions in the history result can be limited using `--max` optio
 
 [source,bash,options="nowrap"]
 ----
-skipper:>history --release-name helloworldlocal --max 2
+skipper:>release history --release-name helloworldlocal --max 2
```

Comment:
I removed `--max` as there was a hard to debug marshalling issue calling it via restTemplate after converting over to use `Resource<Release>`  Will fix on merge.